src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -26,4 +26,16 @@ public Movie get(Long id) {
     public List<Movie> getAll() {
         return movieDao.getAll();
     }
+
+    @Override
+    public Boolean update(Movie movie) {
```

Comment:
The return type of the `update` method should be `boolean` instead of `Boolean` to avoid unnecessary boxing.